site/src/pages/ManagementSettingsPage/OrganizationSettingsPage.tsx

```diff
@@ -0,0 +1,63 @@
+import type { FC } from "react";
+import { useMutation, useQueryClient } from "react-query";
+import { useNavigate } from "react-router-dom";
+import {
+  updateOrganization,
+  deleteOrganization,
+} from "api/queries/organizations";
+import { ErrorAlert } from "components/Alert/ErrorAlert";
+import { displaySuccess } from "components/GlobalSnackbar/utils";
+import { Stack } from "components/Stack/Stack";
+import { useOrganizationSettings } from "./ManagementSettingsLayout";
+import { OrganizationSettingsPageView } from "./OrganizationSettingsPageView";
+
+const OrganizationSettingsPage: FC = () => {
+  const navigate = useNavigate();
+
+  const queryClient = useQueryClient();
+  const updateOrganizationMutation = useMutation(
+    updateOrganization(queryClient),
+  );
+  const deleteOrganizationMutation = useMutation(
+    deleteOrganization(queryClient),
+  );
+
+  const { currentOrganizationId, organizations } = useOrganizationSettings();
+
+  const org = organizations.find((org) => org.id === currentOrganizationId);
+
+  const error =
+    updateOrganizationMutation.error ?? deleteOrganizationMutation.error;
+
+  if (!currentOrganizationId) {
+    return null;
+  }
+
+  if (!org) {
+    return null;
+  }
```

Comment:
I think we can merge the conditions in one:
```ts
if (!currentOrganizationId || !org) {
    return null;
  }
```

Also, when are `currentOrganizationId` or `org` empty?

Reply:
if the organization doesn't exist, or if you're in the deployment settings instead of the org settings.